bundles/org.eclipse.e4.ui.css.core/src/org/eclipse/e4/ui/css/core/utils/ClassUtils.java

```diff
@@ -22,13 +22,7 @@ public class ClassUtils {
 	 * Return the simple name of Class <code>c</code>.
 	 */
 	public static String getSimpleName(Class<?> c) {
-		String name = c.getName();
-		int index = name.lastIndexOf('.');
-		if (index > 0) {
-			name = name.substring(index + 1, name.length());
-		}
-
-		return name.replace('$', '-');
+		return c.getSimpleName();
```

Comment:
will this really replace `$` with `-` as before?

Reply:
good point i will fix that